gcloud/monitoring/timeseries.py

```diff
@@ -123,6 +140,32 @@ def __repr__(self):
         )
 
 
+def _get_typed_value_from_value(value):
```

Comment:
I wonder. Would you perhaps prefer to have a function that actually returns the typed value? For example, given `3.14`, it would return `{'doubleValue': 3.14}`.